iocage/lib/Prompts.py

```diff
@@ -0,0 +1,20 @@
+import helpers
+
+
+class Prompts:
+
+    def __init__(self, host=None):
+        helpers.init_host(self, host)
+
+    @property
+    def release(self):
+        i = 0
+        default = None
+        for available_release in self.host.distribution.releases:
+            if available_release.name == self.host.release_version:
+                default = i
+                print(f"[{i}] \033[1m{available_release.name}\033[0m")
+            else:
+                print(f"[{i}] {available_release.name}")
```

Comment:
This Prompts class is more a part of the cli than the lib and is exclusively used for user interaction. I could not find a better path to save it, but I'm open to move it somewhere else.